src/standalone/Vegetation/canopy_boundary_fluxes.jl

```diff
@@ -47,6 +58,8 @@ struct AtmosDrivenCanopyBC{
     radiation::B
     "Ground conditions"
     ground::G
+    "Surface flux parameterization"
```

Comment:
Maybe this can be clearer: "Surface flux (latent and sensible heat) parameterization"